zboxcore/sdk/chunked_upload_model.go

```diff
@@ -113,6 +116,30 @@ type UploadBlobberStatus struct {
 	UploadLength int64 `json:"upload_length,omitempty"`
 }
 
+type status struct {
+	Hasher       hasher
+	UploadLength int64 `json:"upload_length,omitempty"`
+}
+
+func (s *UploadBlobberStatus) UnmarshalJSON(b []byte) error {
+	//fixed Hasher doesn't work in UnmarshalJSON
+	status := &status{}
+
+	if err := json.Unmarshal(b, status); err != nil {
+		return err
+	}
+
+	status.Hasher.File = sha256.New()
+	status.Hasher.Content.Hash = func(left, right string) string {
```

Comment:
accessing Content.Hash without checking content is nil or not may create nil pointer err.